apps/measures/models.py

```diff
@@ -125,3 +149,191 @@ class MeasuresOverviewPage(MetadataPageMixin, Page):
         index.SearchField('page_intro_de_ls', partial_match=True),
     ]
+
+
+class MeasuresDetailPage(Page):
+    page_title_de = models.CharField(
+        max_length=120, blank=False)
+    page_title_en = models.CharField(
+        max_length=120, blank=True)
+    page_title_de_ls = models.CharField(
+        max_length=120, blank=True)
+
+    responsible_orgs_de = models.CharField(
+        max_length=80, blank=False)
+    responsible_orgs_en = models.CharField(
+        max_length=80, blank=True)
+    responsible_orgs_de_ls = models.CharField(
+        max_length=80, blank=True)
+
+    involved_orgs_de = models.CharField(
+        max_length=300, blank=True)
+    involved_orgs_en = models.CharField(
+        max_length=300, blank=True)
+    involved_orgs_de_ls = models.CharField(
+        max_length=300, blank=True)
+
+    districts = MultiSelectField(
+        blank=True,
+        max_choices=3,
+        choices=BEZIRK_CHOICES
+    )
+
+    date_de = models.CharField(
+        max_length=50, blank=True)
+    date_en = models.CharField(
+        max_length=50, blank=True)
+    date_de_ls = models.CharField(
+        max_length=50, blank=True)
+
+    status = models.CharField(max_length=1, blank=True, choices=STATUS_CHOICES)
+
+    text_image = models.ForeignKey(
+        'apps_images.CustomImage',
+        null=True,
+        blank=False,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+
+    body_de = fields.RichTextField(blank=False)
+    body_en = fields.RichTextField(blank=True)
+    body_de_ls = fields.RichTextField(blank=True)
+
+    body_participation_de = fields.RichTextField(blank=True)
+    body_participation_en = fields.RichTextField(blank=True)
+    body_participation_de_ls = fields.RichTextField(blank=True)
+
+    body_effect_de = fields.RichTextField(blank=True)
+    body_effect_en = fields.RichTextField(blank=True)
+    body_effect_de_ls = fields.RichTextField(blank=True)
+
+    contact_name = models.CharField(max_length=80, blank=False)
+    contact_email = models.CharField(max_length=50, blank=True)
+    contact_phone = models.CharField(max_length=20, blank=True)
+
+    contact_organisation_name_de = models.CharField(max_length=80, blank=False)
+    contact_website_de = models.CharField(max_length=20, blank=True)
+
+    contact_organisation_name_en = models.CharField(max_length=80, blank=True)
+    contact_website_en = models.CharField(max_length=20, blank=True)
+
+    contact_organisation_name_de_ls = models.CharField(
+        max_length=80, blank=True
+    )
+    contact_website_de_ls = models.CharField(max_length=20, blank=True)
+
+    page_title = TranslatedField(
+        'page_title_de',
+        'page_title_en',
+        'page_title_de_ls',
+    )
+
+    responsible_orgs = TranslatedField(
+        'responsible_orgs_de',
+        'responsible_orgs_en',
+        'responsible_orgs_de_ls',
+    )
+
+    involved_orgs = TranslatedField(
```

Comment:
maybe same for below, also as these are seen by editors (who at somepoint may not be max) we can use full length words here